pkg/app/piped/executor/analysis/metrics_analyzer.go

```diff
@@ -354,7 +354,11 @@ func (a *metricsAnalyzer) analyzeWithCanaryPrimary(ctx context.Context) (bool, e
 
 // compare compares the given two samples using Mann-Whitney U test.
 // Considered as failure if it deviates in the specified direction as the third argument.
+// If both of the point values is empty, this returns true.
 func compare(experiment, control []float64, deviation string) (acceptable bool, err error) {
+	if len(experiment) == 0 && len(control) == 0 {
+		return true, nil
+	}
```

Comment:
I think we should have at least a log message on the Stage to let users know.
WDYT?

Reply:
Right, Let me fix it.